README.md

```diff
@@ -13,13 +13,88 @@ If you missed IPFS Camp or want to revisit its content, everything will remain a
+| **Electives**  | | | | |
+| Identity on the DWeb                                               | @satazor @paulobmarcos @joaosantos15 | Understand the state of decentralized identity on the DWeb and current efforts on creating open and interoperable models to solve self-sovereign identities. | [ELECTIVE_COURSE_A](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_A) | soon™ |
+| Managing pinsets with IPFS Cluster                                 | @hsanjuan @lanzafame @meiqimichelle  | A workshop on operating and using IPFS Cluster to orchestrate a common pinset among multiple IPFS nodes | [ELECTIVE_COURSE_B](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_B) | soon™ |
+| TBA (But will have cats with lasers...)                            | @gorhgorhi                           |                | [ELECTIVE_COURSE_C](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_C) | soon™ |
+| Building DApps with Textile, the iCloud for the DWeb               | @balupton @carsonfarmer @andrewxhill |                | [ELECTIVE_COURSE_D](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_D) | soon™ |
+| Deploying IPFS Infrastructure                                      | @mburns @obo20                       | Take your IPFS game to the next level by learning how to deploy and manage your very own IPFS node in the cloud.                                                               | [ELECTIVE_COURSE_E](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_E) | soon™ |
+| Starting, Growing, and Stewarding your IPFS Community              | @nukemandan @kk3wong @stefanhans     | Let's get Meta! A workshop on effective workshops! And how to foster a community                                                                | [ELECTIVE_COURSE_F](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_F) | soon™ |
+| Managing Datasets with QRI                                         | @b5                                  | An open source tool for versioned, scriptable, exportable, collaborative datasets | [ELECTIVE_COURSE_G](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_G) | soon™ |
+| Making the Browser a True User Agent                               | @autonome @Gozala @soapdog           | We’ll show how browsers act as a value filter on our world, teach some approaches to making the browser truly be an agent that represents you, and break into groups to start building our own true user agents.| [ELECTIVE_COURSE_H](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_H) | soon™ |
+
+### [📃 Poster Projects](POSTER_PROJECTS)
+
+TODO
+
+### [🐋 Deep Dives](DEEP_DIVES)
```

Comment:
@alanshaw can you help me understand which Deep Dives happened? I want to link them form the main README and then soon link the recording too

Reply:
All of them listed here happened https://github.com/ipfs/camp/tree/master/DEEP_DIVES#topics apart from 23 - connection management.

Reply:
Thank you Sir! :)